modules/preferences/main/index.coffee

```diff
@@ -287,7 +287,13 @@ class Preferences extends hx.EventEmitter
   applyTimezoneOffset: (date, offset) ->
     offset ?= @_.timezoneOffsetLookup(@timezone(), date.getTime()) || 0
     utc = date.getTime() + (date.getTimezoneOffset() * 60000)
-    new Date(utc + offset * 60 * 60 * 1000)
+    # In fact timezoneOffsetLookup working properly and returns
+    # 2 for Sofia but the proper offset is not this because
+    # If we want to get real date we must subtract
+    # these hours - the real offset for Sofia for
+    # example is -120
+    realOffset = -1 * offset * 60
```

Comment:
Try to run the tests. If returned offset was in minutes the below multiplication will be enough.